README.md

```diff
@@ -1,20 +1,19 @@
 # Introduction 
-Data service micro-service helps TDEI system to query information specific to the OSW.
+Data service micro-service helps TDEI system to query & persist the information specific to the OSW.
```

Comment:
This service helps in persisting & qyerying information related to OSW data.
Cannot query before persisting. That's the idea